Library/Formula/imgur-screenshot.rb

```diff
@@ -0,0 +1,23 @@
+class ImgurScreenshot < Formula
+  desc "Take screenshot selection, upload to imgur. + more cool things"
+  homepage "https://github.com/jomo/imgur-screenshot"
+  head "https://github.com/jomo/imgur-screenshot.git"
+  url "https://github.com/jomo/imgur-screenshot/archive/v1.5.4.tar.gz"
+  sha256 "147fe1c3b3e407d952fa8abbdb9b436192133cf7a56278edc0dd87c2a9c9c430"
+
+  option "with-terminal-notifier", "terminal-notifier is needed for Mac OS X User Notifications"
```

Comment:
Na, It's fine. Just wasn't sure if the Mac OS X part was redundant since that's the platform all Homebrew uses will be running. Let's de-dupe the readout though, so it just reads:

`"with-terminal-notifier", "Needed for OS X User Notifications"` or something.